dwallet/src/lib.rs

```diff
@@ -0,0 +1,5 @@
+pub mod light_client;
+mod utils;
+
+pub use consensus::{rpc, ConsensusStateManager};
+pub use execution::verify_proof;
```

Comment:
```suggestion
```
Export these from their corresponding crates, no need to export from `dwallet`, it's a bit confusing.